src/index.html

```diff
@@ -16,17 +16,29 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      class="searchbar"
+      data-qa="search-form"
+    >
+      <input
+        class="searchbar__input searchbar__input-big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
+      data-qa="search-input"
+      class="searchbar"
+    >
+      <input
+        class="searchbar__input searchbar__input-small"
+        type="text"
+        data-qa="hover"
```

Comment:
The `data-qa` attribute `hover` might not be correctly set according to the task requirements. Please verify and adjust as needed.